### PR TITLE
Apply some more suggestions

### PR DESCRIPTION
**Pull request by ktmf01**
This should be the last PR at least until the issues sent on the mail resolve.

### CODE REVIEW

File: flac.md [hunk 1]
```diff
@@ -135,9 +159,17 @@ The FLAC format uses two forms of Rice coding, which only differ in the number o
...
+Predictor coefficient precision bits 0b1111 | [Linear predictor subframe](#linear-predictor-subframe)
 
-All numbers used in a FLAC bitstream MUST be integers, there are no floating-point representations. All numbers MUST be big-endian coded, except the field length used in Vorbis comments, which MUST be little-endian coded. All numbers MUST be unsigned except linear predictor coefficients, the linear prediction shift and numbers which directly represent samples, which MUST be signed. None of these restrictions apply to application metadata blocks or to Vorbis comment field contents.
+All numbers used in a FLAC bitstream are integers, there are no floating-point representations. All numbers are big-endian coded, except the field length used in Vorbis comments (see [Vorbis comment metadata block](#vorbis-comment)), which are little-endian coded. All numbers are unsigned except linear predictor coefficients, the linear prediction shift (see [lineair predictor subframe](#linear-predictor-subframe)) and numbers which directly represent samples, which are signed. None of these restrictions apply to application metadata blocks or to Vorbis comment field contents.
```
Comment (retokromer) [inline flac.md:172]: typo: lineair > linear

File: flac.md [hunk 2]
```diff
@@ -197,7 +229,7 @@ Value   | Metadata block type
...
+If the streaminfo metadata block contains incorrect or incomplete information, decoder behaviour is left unspecified (i.e. up to the decoder implementation). A decoder MAY choose to stop further decoding when the information supplied by the streaminfo metadata block turns out to be incorrect or invalid. A decoder accepting information from the streaminfo block (most significantly the maximum frame size, maximum block size, number of audio channels, number of bits per sample and total number of samples) without doing further checks during decoding of audio frames could be vulnerable to buffer overflows. See also [the section on security considerations](#security-considerations).
```
Comment (retokromer) [inline flac.md:232]: use US spelling: behaviour > behavior

File: flac.md [hunk 3]
```diff
@@ -814,9 +856,46 @@ The following information serves as the registration form for the "audio/flac" m
...
+- And of course, Claude Shannon, see [@Shannon].
+
+<reference anchor="Rice" target="https://ieeexplore.ieee.org/document/1090789">
+    <front>
+        <title>Adaptive Variable-Length Coding for Efficient Compression of Spacecraft Television Data</title>
+        <author initials="RF" surname="Rice" fullname="Robert Rice">
+            <organisation>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA, USA</organisation>
+        </author>
+        <author initials="JR" surname="Plaunt">
+            <organisation>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA, USA</organisation>
```
Comment (retokromer) [inline flac.md:871]: use US spelling: organisation > organization

File: flac.md [hunk 4]
```diff
@@ -814,9 +856,46 @@ The following information serves as the registration form for the "audio/flac" m
...
+    </front>
+    <seriesInfo name="DOI" value="10.1109/TCOM.1971.1090789"/>
+</reference>
+
+<reference anchor="Durbin" target="https://www.jstor.org/stable/1401322">
+    <front>
+        <title>The Fitting of Time-Series Models </title>
+        <author initials="J" surname="Durbin" fullname="James Durbin">
+            <organisation>University of North Carolina</organisation>
+            <organisation>University of London</organisation>
```
Comment (retokromer) [inline flac.md:883]: ditto

File: flac.md [hunk 5]
```diff
@@ -814,9 +856,46 @@ The following information serves as the registration form for the "audio/flac" m
...
+        <date month="12" year="1959"/>
+    </front>
+    <seriesInfo name="DOI" value="10.2307/1401322"/>
+</reference>
+
+<reference anchor="Shannon" target="https://ieeexplore.ieee.org/document/1697831">
+    <front>
+        <title>Communication in the Presence of Noise</title>
+        <author initials="CE" surname="Shannon" fullname="Claude Shannon">
+            <organisation>Bell Telephone Laboratories, Inc., Murray Hill, NJ, USA</organisation>
```
Comment (retokromer) [inline flac.md:894]: ditto

**Review by retokromer — COMMENTED** [review]
a few nits

File: flac.md (hunk 3 above)
Comment (ktmf01) [reply flac.md:871]: Strange that xml2rfc didn't complain about this

Comment (retokromer) [reply flac.md:871]: (I personally prefer the UK spelling. But it should be consistent either way.)

File: flac.md (hunk 1 above)
Comment (ktmf01) [reply flac.md]: Funny, that's the Dutch spelling. Thanks for checking!

**Review by retokromer — APPROVED** [review]

**Pull request merged**